anyblok_pyramid/anyblok.py

```diff
@@ -98,9 +99,9 @@ def savepoint(self):
             raise AttributeError("savepoint")
         return self._savepoint
 
-    def _savepoint(self):
-        self.registry.System.Cache.clear_invalidate_cache()
-        return AnyBlokSessionSavepoint(self.registry)
+    # def _savepoint(self):
+    #     self.registry.System.Cache.clear_invalidate_cache()
+    #     return AnyBlokSessionSavepoint(self.registry)
```

Comment:
Not sure how to get proper anyblok registry here

Reply:
https://github.com/AnyBlok/AnyBlok/pull/242